guide/english/cplusplus/vector/index.md

```diff
@@ -233,3 +233,50 @@ int main(){
 ```
 
 You can also sort in descending using lamda like the one above.
+
+### Finding an element in a ```std::vector```
+Before C++11:
+```c++
+#include <iostream>
```

Comment:
Running this code produces the following error: `error: no matching function for call to 'find'`